### ISSUE TITLE
unable to phase through structures as revenant when spawning at AI

### ISSUE THREAD

**Issue by feenry** [space-wizards/space-station-14#31674]
## Description
Revenant that spawns at/near AI will spawn outside the station in space AND removes the ability for revenant to phase through structures like windows, walls, doors, etc.

**Reproduction**
N/A

**Screenshots**

**Additional context**
After ahelping and being teleported inside the station near science, I was still not able to phase through structures. Only after I attempted(and failed from being hit by evil doctors) gathering essence from a player was I able to phase through structures again.

**Comment by slarticodefast** [comment]
Duplicate of #31568
I don't think the spawn location was relevant for this bug, it just had the wrong fixture, which got fixed now.
